### PR TITLE
Disable chat during night and fix sending white spaces

### PR DESCRIPTION
**Pull request by zhangshyue**

### CODE REVIEW

File: ubg/src/pages/Room.js [hunk 1]
```diff
@@ -413,7 +413,8 @@ function Room({setUsersData, setCurrentUser, setRoomData}) {
                       </div>
                       <Chat
                         disabled={(mafiaChatSelected && roomData.day) ||
-                          !inGame}
+                          !inGame ||
+                          (!mafiaChatSelected && !roomData.day)}
```
Comment (shandyk18) [inline ubg/src/pages/Room.js:417]: Can you simplify it to `!inGame || !roomData.day`?

Comment (shandyk18) [inline ubg/src/pages/Room.js:417]: Will this disable the normal chat for mafia during the night as well?

File: ubg/src/pages/Room.js [hunk 2]
```diff
@@ -413,7 +413,8 @@ function Room({setUsersData, setCurrentUser, setRoomData}) {
                       </div>
                       <Chat
                         disabled={(mafiaChatSelected && roomData.day) ||
-                          !inGame}
+                          !inGame ||
```
Comment (shandyk18) [inline ubg/src/pages/Room.js:416]: Will this disable the normal chat for mafia during the night as well?

Comment (zhangshyue) [reply ubg/src/pages/Room.js:416]: I don't think so cuz !mafiaChatSelected make sure that it's not mafia chat

**Pull request merged**